external/protobuf/CMakeLists.txt

```diff
@@ -48,5 +48,19 @@ if (EMIL_HOST_BUILD)
         endforeach()
     endfunction()
 
+    function(remove_mp)
```

Comment:
What about merging remove_mp into add_protobuf_target_properties, and renaming that to modify_protobuf_target_properties?